### PR TITLE
feat: [documentation-v2] - breaking changes and available components …

### PR DESCRIPTION
**Pull request by mattgoud**
…(wip)

### ❓ Types of changes

- [X] 📖 Documentation (updates to the documentation or readme)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality)
- [ ] 📦 New feature (a non-breaking change that adds functionality)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📝 Checklist

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes
- [ ] The component exists on old Prestashop UIKit and my pull request on [migrating documentation](https://github.com/PrestaShopCorp/devdocs.uikit.prestashop.com) is accepted.

### CODE REVIEW

File: RELEASE-NOTES-V2.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+# Puik v2 (Beta) - Breaking Changes and Available Components
+
+## Breaking Changes
+
+In the transition from Puik v1 to Puik v2, we've made several significant changes to improve the library's performance, usability, and maintainability. Here are the key changes that you should be aware of:
```
Comment (cnavarro-prestashop) [inline RELEASE-NOTES-V2.md:5]: just "we made"

File: RELEASE-NOTES-V2.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+- **Monorepo Structure**: In Puik v2, we've transitioned to a monorepo structure. This structure includes separate libraries for Vue 3 components, Web components, a component resolver for our Vue 3 component library, a CSS library containing all the classes used in our components, a Tailwind CSS preset that contains all the design tokens used to create the components. This change was made to improve the modularity and maintainability of the project.
```
Comment (cnavarro-prestashop) [inline RELEASE-NOTES-V2.md:7]: "we moved to" instead "we've transitioned" + "and a Tailwind CSS " instead ", a Tailwind CSS"

File: RELEASE-NOTES-V2.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+  
+  :warning: **Please note that this new structure may affect the import paths for the Puik library in your project. We recommend checking the README files for each package to understand the correct import paths.**
+
+- **Snackbar Component**: In Puik v1, the Snackbar component was implemented using Headless UI. In Puik v2, we've transitioned to using Radix Vue for the Snackbar component. This change was made to improve the component's accessibility and flexibility. If you're using the Snackbar component in your project, please refer to the Storybook documentation for details on how to adapt your code to this change.
```
Comment (cnavarro-prestashop) [inline RELEASE-NOTES-V2.md:11]: "we moved to" instead "transitioned"

**Review by cnavarro-prestashop — APPROVED** [review]
Ok just 2 comments on typo

File: RELEASE-NOTES-V2.md (hunk 2 above)
Comment (mattgoud) [reply RELEASE-NOTES-V2.md:7]: ok

**Pull request merged**